test/cli.js

```diff
@@ -42,24 +42,6 @@ describe('CLI:', function() {
     })
   })
 
-  describe('pipe', function() {
```

Comment:
So the ability to pipe styledown is removed?

Reply:
Yes. It's a feature that we don't use (at the moment the cli is not used at all 😅) and getting rid of the pipe feature will make it easier to remove the package `read-input` later (last release 5 years ago). But if you don't agree we can keep this feature

Reply:
@ryuran ?

Reply:
OK